presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -55,7 +55,8 @@ Date and Time Functions
 
 .. data:: current_timestamp
 
-    Returns the current timestamp with time zone as of the start of the query.
+    Returns the current timestamp(3) with time zone as of the start of the
```

Comment:
This function now allows the from `current_timestamp(precision)`, similar to `localtimestamp(precision)`

Reply:
```
presto:tiny> select current_timestamp(6);
                     _col0
------------------------------------------------
 2020-06-24 08:25:31.759993 America/Los_Angeles
(1 row)
```

Reply:
This is currently a phrase, not a type, so it shouldn't have a precision. Let's do it like `localtimestamp` where we say

> , with `3` digits of subsecond precision.

Reply:
We'll need to add a `current_timestamp(p)` function as @martin mentions. Copy `localtimestamp` as much as possible.

Reply:
All done